compiler/testData/ir/irText/declarations/annotations/primaryConstructorParameterWithAnnotations.fir.txt

```diff
@@ -25,8 +25,6 @@ FILE fqName:<root> fileName:/primaryConstructorParameterWithAnnotations.kt
         DELEGATING_CONSTRUCTOR_CALL 'public constructor <init> () [primary] declared in kotlin.Any'
         INSTANCE_INITIALIZER_CALL classDescriptor='CLASS CLASS name:Test modality:FINAL visibility:public superTypes:[kotlin.Any]'
     PROPERTY name:x visibility:public modality:FINAL [val]
-      annotations:
-        Ann
```

Comment:
Note that this removal is legitimate, since the constructor above has the annotation already (for `class Test(@param:Ann val x`)`. All other IR dumps show that annotations are now associated with correct targets.

Reply:
Yes, I know that some annotations are at wrong targets now